WorkflowLayer/Parameters/STARAlignmentParameters.cs

```diff
@@ -44,6 +56,6 @@ public class STARAlignmentParameters
         public List<string> ChimericSamFiles { get; set; }
         public List<string> ChimericJunctionFiles { get; set; }
         public bool UseReadSubset { get; set; }
-        public int ReadSubset { get; set; }
+        public int ReadSubset { get; set; } = 300000;
```

Comment:
Let's use this type of initializer, rather than the blank constructor, in the future